ionic_user_interface/src/views/Signup.vue

```diff
@@ -70,6 +70,19 @@
                 >
                   Sign up
                 </ion-button>
+                <div class="privacy-policy">
+                  <ion-checkbox v-model="termsCheckbox" />
+                  <ion-text>
+                    I agree to
+                    <a href="https://routemaker.rocks/privacy.html" target="_blank">
```

Comment:
Maybe just href="/privacy.html"? Also, we might wanna consider putting this as an inline scrollable component like other websites

Reply:
If we change to native, it won't work anymore right? Cos native no domain

Reply:
Ah right, ok I think maybe we should just add the terms and condition as an inline scrollable div or smth